python/lsst/pipe/supertask/activator.py

```diff
@@ -292,7 +321,11 @@ def load_super_task(super_taskname):
 
         if super_task_module:
             # Need to add a try -- except here
-            python_mod_stask = importlib.import_module(package.__name__ + '.' + super_task_module)
+            if hasattr(package, 'mock'):
+                (file_path, filename, data) = imp.find_module(super_task_module, package.__path__)
+                python_mod_stask = imp.load_module(super_task_module, file_path, filename, data)
```

Comment:
This code should work on Python 3 (by summer) so it can't use `imp`. I'm not sure how we can have a race condition in the code since it's single threaded. Prepending to `sys.path` seems to be the only portable option. There is a risk that they have a typo in the module that leads to another package being loaded. I guess you can check the path `commonprefix` against `__file__` after you import it if you want to be sure it was read from the right place.